### PR TITLE
feat(calling): show double tap indicator once per application lifetime (WPB-314)

### PR DESCRIPTION
**Pull request by ohassine**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-314" title="WPB-314" target="_blank">StoryWPB-314</a>  Indicate to the user that they can double click to maximise the video stream
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [x] contains a reference JIRA issue number like `SQPIT-764`
    - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

double tap indicator is displayed every time we open ongoing call screen

### Solutions

Change it to be displayed only once for the current user per application lifetime.

Needs releases with:

- [ ] GitHub link to other pull request

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

#### How to Test

1. Answer incoming call.
2. You should see the indicator
3. Second time you answer a call you should not see it
PS: If you switch to a different user you should see for the fist call as well

----
#### PR Post Submission Checklist for internal contributors (Optional)

- [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

- [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [191](https://10.10.124.134/job/AR-build-pipeline/191/) **failed**.

**Comment by AndroidBob** [comment]
Build [197](https://10.10.124.134/job/AR-build-pipeline/197/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.3.0(6951013)(6951013)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.3.0(6951013)(6951013)-staging-compat.apk)

File: app/src/main/kotlin/com/wire/android/ui/calling/ongoing/OngoingCallViewModel.kt [hunk 1]
```diff
@@ -70,7 +77,13 @@ class OngoingCallViewModel @Inject constructor(
...
             }
         }
-        showDoubleTapToast()
+        viewModelScope.launch {
+            delay(DELAY_TO_SHOW_DOUBLE_TAP_TOAST)
+            shouldShowDoubleTapToast = globalDataStore.getShouldShowDoubleTapToast(currentUserId.toString())
+            if (shouldShowDoubleTapToast) {
+                startDoubleTapToastDisplayCountDown()
+            }
+        }
```
Comment (alexandreferris) [inline app/src/main/kotlin/com/wire/android/ui/calling/ongoing/OngoingCallViewModel.kt]: Couldn't this still be done inside the function that was removed? `showDoubleTapToast`

**Review by alexandreferris — APPROVED** [review]
Awesome! Just have a tiny comment but mostly asking about it

**Comment by AndroidBob** [comment]
Build [214](https://10.10.124.134/job/AR-build-pipeline/214/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.3.0(6959680)(6959680)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.3.0(6959680)(6959680)-staging-compat.apk)

**Review by Garzas — APPROVED** [review]

**Review by vitorhugods — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [259](https://10.10.124.134/job/AR-build-pipeline/259/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.3.0(6976382)(6976382)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.3.0(6976382)(6976382)-staging-compat.apk)

**Pull request merged**